serviceaccounts.mjs

```diff
@@ -0,0 +1,70 @@
+import * as fs from 'fs';
```

Comment:
The existing equivalent ruby file still exist, this is an attempt to convert that file in javascript